layers/+lang/reason/README.org

```diff
@@ -0,0 +1,70 @@
+#+TITLE: Reason layer
+
+[[https://reasonml.github.io/img/reason.svg]]
+
+* Description
+
+Spacemacs layer for ReasonML, based around [[https://github.com/reasonml-editor/reason-mode][reason-mode]].
+
+** Features:
+  - Syntax highlighting / indentation
+  - Autocomplete (via `merlin`)
```

Comment:
The markdown inline code (surrounding backquotes) syntax needs to be changed to org verbatim (surrounding equal signs), throughout the readme.org file.
#### before:
`` `merlin` ``
#### after:
`=merlin=`